core/trino-main/src/test/java/io/trino/operator/aggregation/BenchmarkDecimalAggregation.java

```diff
@@ -108,36 +130,66 @@ public Block benchmarkEvaluateFinal(BenchmarkData data)
         @Param({"10", "1000"})
         private int groupCount = 10;
 
+        @Param({"SHORT", "LONG", "MIXED"})
+        private String decimalSize = "SHORT";
+
+        @Param({"true", "false"})
```

Comment:
How does group ordering matter?

Reply:
> For the partial aggregation adaptation case, the group ids are always consecutive.

This has little to do with serialization.

> It doesn't matter for serialization, but for evaluateIntermediate the main bottleneck is actually jumping to random memory locations in case groups are located randomly.

For skip-cases groups are consecutive and number of them is small, so it probably doesn't matter too.